molgenis/molgenis-emx2
# fix: don't remove filters from store. closes #3087

fixes: https://github.com/molgenis/molgenis-emx2/issues/3087

What are the main changes you did:
- the filters are cleared from there store if they are no longer in the url.. this makes using a store useless, and we want to have the state saved.

how to test:
- Set filters
- Go in a biobank
- Go back to the main screen
- Expect the filters to still be set
